tests/conftest.py

```diff
@@ -5,6 +5,23 @@
 from workoutizer import settings as django_settings
 
 
+def pytest_addoption(parser):
+    parser.addoption(
+        "-B",
+        "--browser",
+        dest="browser",
+        action="store",
+        default="firefox",
+        help="Add Browsers used for the test, Firefox is enabled by default, Chrome can be added",
+    )
+
+
+@pytest.fixture
+def browser(request):
+    "pytest fixture for browser"
+    return request.config.getoption("-B")
+
+
```

Comment:
Nice, I never had a look at how to implement additional pytest arguments 🙂 looks super straight forward, good job!

Reply:
Yes this is pretty straightforward, I had a bit more difficulty to understand the parameterizing with `metafunc` though. But that is a future problem :wink: